xla/service/gpu/fusions/cudnn.cc

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2024 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "xla/service/gpu/fusions/cudnn.h"
+
+#include "absl/strings/escaping.h"
+#include "xla/hlo/ir/hlo_instructions.h"
+#include "xla/service/gpu/runtime/cudnn_thunk.h"
+#include "xla/service/llvm_ir/llvm_util.h"
+
+namespace xla {
+namespace gpu {
+
+absl::StatusOr<FusionEmissionResult> CuDnnFusion::Emit(
+    IrEmitterContext& ir_emitter_context, mlir::lmhlo::FusionOp fusion_op,
+    const HloFusionInstruction& fusion) const {
+#if GOOGLE_CUDA
+  if (ir_emitter_context.emit_ir_from_hlo()) {
```

Comment:
This is actually a constant true at this point. Can you just remove the checks here? I'll clean it up if I remember.

Reply:
Done.